### PR TITLE
Navigation pattern page in site

### PR DESCRIPTION
**Pull request by Fercas123**

### CODE REVIEW

File: site/docs/patterns/navigation.mdx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  ]}
+  label="Show underlying grid"
+/>
+
+<ExampleContainer type="positive">
+  - Use a layout grid for the full width of your app’s resolution if it is not responsive, or just for the content area if it is.
+  - Use [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) to determine how the navigation will display at various viewport sizes.
+</ExampleContainer>
+
+## Expanding the pattern
```
Comment (origami-z) [inline site/docs/patterns/navigation.mdx:178]: Right hand nav is a bit awkward on the last heading, not sure why indented

**Review by origami-z — APPROVED** [review]

File: site/docs/patterns/navigation.mdx (hunk 1 above)
Comment (Fercas123) [reply site/docs/patterns/navigation.mdx:178]: might be from the previous build, it was a ### and i just fixed it

Comment (Fercas123) [reply site/docs/patterns/navigation.mdx:178]: but I'll keep an eye to make sure it's fixed

Comment (Fercas123) [reply site/docs/patterns/navigation.mdx:178]: fixed :)

**Review by JoanaMMoreira — APPROVED** [review]

**Pull request merged**